src/igraph_gmp.h

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Let's add the usual license header here; just copy it from one of the other source files and update the copyright years and the name if needed (use "The igraph development team" for the name).